src/wrappers/AuthorizableWrapper.ts

```diff
@@ -0,0 +1,52 @@
+/*
+  Copyright 2018 Set Labs Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+'use strict';
+
+import Web3 from 'web3';
+
+import { ContractWrapper } from '.';
+import { BigNumber } from '../util';
+import { Address } from '../types/common';
+
+/**
+ * @title  AuthorizableWrapper
+ * @author Set Protocol
+ *
+ * The AuthorizableWrapper handles all functions and states related to authorizable contracts
+ *
+ */
+export class AuthorizableWrapper {
```

Comment:
Why do we need a generalized authorizable wrapper? Are you going to add set owner on here as well?

Reply:
Each of the contracts that extend Authorizable should have their own `getAuthorizedAddresses` address in the abi

Reply:
Hmm... that's true. I'll get rid of it.